README.md

```diff
@@ -1,2 +1,65 @@
 # Weather-app
-This project is building a weather app to search for temperature at any place in the world, using weather API and Giphy API and make handy use of promises  or preferably async/await functions. 
+This project is building a weather app to search for temperature at any place in the world, using weather API and make handy use of promises  or preferably async/await functions. 
+
+## Description 
+The app uses an async function which takes the location name as an argument, and fetches weather data from openweather API. 
+An await fetch function is used to fetch the weatherdata from the API and the response is assigned to a variable.
+The await fetch function accepts the API url whose path is an constructed from, a location name and key accessed through subscription. 
+
+### Form 
+The from has a text input element and a submit button. A submit event listner function takes the text input and assigns it to the location. Then it calls the async function(weatherCondition) which takes the location as argument.
+
+### The Weather Div Element 
+The weather Div element is the wrapper element of the displays of the weather condition. 
+It displays temperature, sky condition, feel_like temperature and the weather icon. 
+
+### The aysnc - 'weatherCondition(location)' function
+The watherCondition function is an async function which has amn await fetch function inside it. It fetches API data, assignes values to a weather objecta and calls the 'showWeather' function to display the weather data in the browser. 
+
+### The showWeather function
+The showWeather function accepts weather object as an argument and assignes the values as innerText to the respective HTML elements of the HTML skeleton. 
+
+### Dynamic css
+The showWeather function dynamically changes the background and the weather icon images based on the fetched weather condition. 
+
+### Demo 
+- Cloudy weather
+![screenshot](src/docs/Screenshot1.png)
+- Sunn weather
+![screenshot](src/docs/Screenshot2.png)
+- Error message 
+![screenshot](src/docs/Screenshot3.png)
+
+## Built With
+
+- Javscript
+- HTML/CSS
+- Webpack 
+- npm 
+- package.json
+- openweather API 
+
+## Live Link
```

Comment:
- [ ] [REQUIRED] Looks like you forget to add the URL of your hosted app. You can either add the URL (to make your repo professional) or take off the Live link section.

Reply:
Hi @krys2fa, thank you for your helpful comments. For the temperature toggle, the app is working as intended. You may have a look at the gif I have attached while I was toggling the temperature between Celsius and Fahrenheit scales, in the browser app. You can also check it by clicking on the temperature. 
For the live link, I will work on it 

Regards,

![chrome-capture](https://user-images.githubusercontent.com/64894927/112719183-a70ba280-8f08-11eb-8f7c-b45a0eeb9f77.gif)